app/src/components/fields/MultiAutocompleteField.tsx

```diff
@@ -23,14 +22,26 @@ export interface IMultiAutocompleteField {
   handleSearchResults?: (input: string) => Promise<void>;
 }
 
+export const handleSortSelectedOption = (
```

Comment:
Can you add a JSDOC description for this function? Just because its a bit of a unique business case to sort the list by appending the selected options to the top.

It might be good to include `autocomplete` in the name of this function somewhere, just because its being exported for use in other components. Maybe something like `sortAutocompleteOptions`?